### PR TITLE
Add recipe and license for SDF.

### PR DESCRIPTION
**Pull request by t-sommer**

### CODE REVIEW

File: recipes/sdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{% set name = "sdf" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps sdf=={{ version }}
```
Comment (CJ-Wright) [inline recipes/sdf/meta.yaml]: Would it be possible to build from a tarball?

File: recipes/sdf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set name = "sdf" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps https://github.com/ScientificDataFormat/SDF-Python/archive/v{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipes/sdf/meta.yaml]: Please structure this like the example
```
source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256 }}
```
The build sequence will download the file so you can pip install from the local copy

**Comment by t-sommer** [comment]
Is there anything else you want changed?

File: recipes/sdf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/SDF-{{ version }}.tar.gz  
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/sdf/meta.yaml]: Could this be a noarch package?

Comment (t-sommer) [reply recipes/sdf/meta.yaml]: With 'noarch: python' set the build complains about the native libraries contained in the package. The Python code itself does not depend on the Python version or platform but one module uses native code (loaded with ctypes).

Comment (CJ-Wright) [reply recipes/sdf/meta.yaml]: Ah oh well

Comment (CJ-Wright) [reply recipes/sdf/meta.yaml]: feel free to revert (I'll merge once you do)

Comment (CJ-Wright) [reply recipes/sdf/meta.yaml]: I didn't realize that you had C code, do you compile it during the build process?

Comment (t-sommer) [reply recipes/sdf/meta.yaml]: The GitHub repository (https://github.com/ScientificDataFormat/SDF-Python) contains pre-built versions of the shared libs for all supported platforms, so the universal wheel can be built on each platform. Build scripts for all supported platforms are also contained.

**Comment by t-sommer** [comment]
Can you merge?

**Pull request merged**